packages/browser-destinations/destinations/jimo/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "@segment/analytics-browser-actions-jimo",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/segmentio/action-destinations",
+    "directory": "packages/browser-destinations/destinations/jimo"
+  },
+  "main": "./dist/cjs",
+  "module": "./dist/esm",
+  "scripts": {
+    "build": "yarn build:esm && yarn build:cjs",
+    "build:cjs": "tsc --module commonjs --outDir ./dist/cjs",
+    "build:esm": "tsc --outDir ./dist/esm"
+  },
+  "typings": "./dist/esm",
+  "dependencies": {
+    "@segment/browser-destination-runtime": "^1.4.0"
```

```suggestion
    "@segment/browser-destination-runtime": "^1.14.0"
```

This version needs to pinned to latest runtime version. There's an open PR that'll make sure that every new scaffolded destination will have the latest version to begin with.

https://github.com/segmentio/action-destinations/blob/main/packages/browser-destination-runtime/package.json#L3